src/python.rs

```diff
@@ -345,6 +437,96 @@ impl Backend {
 		let contents = Cow::from(rope.clone());
 		let contents = contents.as_bytes();
 		let root = some!(top_level_stmt(ast.root_node(), offset));
+
+		// Parse imports from the current file
+		let imports = self.parse_imports(contents).unwrap_or_default();
+		debug!("Parsed imports: {:?}", imports);
+
+		// Check if cursor is on an imported identifier
+		if let Some(cursor_node) = ast.root_node().descendant_for_byte_range(offset, offset) {
+			if cursor_node.kind() == "identifier" {
```

Comment:
We can use a let-chain here
```suggestion
		if let Some(cursor_node) = ast.root_node().descendant_for_byte_range(offset, offset) && cursor_node.kind() == "identifier" {
```